utilities/migration_required

```diff
@@ -2,7 +2,10 @@
 require 'obs_deploy'
 
 host = ARGV[0]
-checkdiff_params = {}
+checkdiff_params = { server: 'https://api.opensuse.org',
```

Comment:
The line 14 in this file needs to be changed to:

```
check_diff = ObsDeploy::CheckDiff.new(**checkdiff_params)
```

Otherwise, this script is crashing with:

```
/home/dany/ansible-obs/vendor/bundle/ruby/3.1.0/bundler/gems/obs_deploy-3fd202e748c5/lib/obs_deploy/check_diff.rb:5:in `initialize': wrong number of arguments (given 1, expected 0) (ArgumentError)
	from utilities/migration_required:14:in `new'
	from utilities/migration_required:14:in `<main>'
```

Reply:
I pushed the fix.